record/index.html.in

```diff
@@ -56,7 +56,7 @@
                 <div uib-accordion-group class="related-table-heading" id="rt-heading-{{::ctrl.makeSafeIdAttr(relatedRef.displayname.value)}}" ng-repeat="relatedRef in relatedReferences track by $index" ng-if="ctrl.showRelatedTable($index)" is-open="tableModels[$index].open">
                     <uib-accordion-heading>
                         <i class="pull-left glyphicon" style="margin-right: 5px;" ng-class="{'glyphicon-chevron-down': tableModels[$index].open, 'glyphicon-chevron-right': !tableModels[$index].open}"></i>
-                        <span ng-if="::relatedRef.displayname.isHTML" ng-bind-html="::relatedRef.displayname.value"></span><span ng-if="::!relatedRef.displayname.isHTML" ng-bind="::relatedRef.displayname.value"></span> ({{::tableModels[$index].rowValues.length}}{{::tableModels[$index].hasNext ? '+' : ''}})
+                        <span ng-if="::relatedRef.displayname.isHTML" ng-bind-html="::relatedRef.displayname.value"></span><span ng-if="::!relatedRef.displayname.isHTML" ng-bind="::relatedRef.displayname.value"></span> (Showing first {{::tableModels[$index].rowValues.length}}{{::tableModels[$index].hasNext ? '+' : ''}} {{::tableModels[$index].rowValues.length>1 ? ' results' : ' result'}})
```

Comment:
It looks like you are adding 2 whitespaces between the number of results and the string `result`/`results`.